Utils/GetLocales.js

```diff
@@ -0,0 +1,13 @@
+const locales = {
+    en: () => import("../locales/en.json").then((r) => r.default),
+    pt: () => import("../locales/pt.json").then((r) => r.default)
+  };
+  
+  export const getLocales = (lang) => {
```

Comment:
i would change this name, i understood this function will return my locale(e.g: br or us).

maybe `getTranslation()` 🤔